openpype/pipeline/colorspace.py

```diff
@@ -221,8 +323,46 @@ def get_data_subprocess(config_path, data_type):
             "config", data_type,
             "--in_path", config_path,
             "--out_path", tmp_json_path
+        ]
+        log.info("Executing: {}".format(" ".join(args)))
```

Comment:
Even though this function is deprecated you've basically duplicated the inner logic here into the `get_wrapped_with_subprocess` function down below so I'd recommend at least cleaning up this code to to start the re-use.